examples/ginkgo_overhead/ginkgo_overhead.cpp

```diff
@@ -72,7 +71,8 @@ env LD_LIBRARY_PATH=.:${LD_LIBRARY_PATH} ./ginkgo_overhead
 #include <iostream>
 
 
-[[noreturn]] void print_usage_and_exit(const char *name) {
+[[noreturn]] void print_usage_and_exit(const char *name)
```

Comment:
not sure why this change was triggered.

Reply:
It's a different version of `clang-format`. I have 7.0.1-8, and I believe Fritz has an older one which indents differently. If you look at the file's history, you find this before my commit:
https://github.com/ginkgo-project/ginkgo/commit/4bec38fa133878f2fdcabd51c4009b4452a396c6#diff-b53488febfc45d801fafbd3a8b7081e4

Reply:
This was probably triggered because not everyone has the same clang-format version (it was changed to the current `develop` state by @fritzgoebel since he has `clang-format` version `6.x`, and most of us have `7.0.1`).